pandas/_libs/parsers.pyx

```diff
@@ -178,13 +178,13 @@ cdef extern from "parser/tokenizer.h":
         char thousands
 
         int header # Boolean: 1: has header, 0: no header
-        int header_start # header row start
-        int header_end # header row end
+        int64_t header_start # header row start
+        int64_t header_end # header row end
```

Comment:
I'm going to let OCD take over and suggest that you add a couple more spaces to align with the hashtag above. :smile: